paddle/framework/scope.cc

```diff
@@ -13,6 +13,8 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #include "paddle/framework/scope.h"
```

Comment:
an empty line under `#include "paddle/framework/scope.h"`
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

change across all the files

Reply:
done